documentation/docs/libraries/nodejs/exchange_guide.md

```diff
@@ -0,0 +1,238 @@
+
+Since addresses are reusable, they can be mapped to your users in a clear and concise way:
+
+- Create an account for every user -> `Multi Account` approach.
+- Create one account with many addresses -> `Single account` approach.
+
+The library supports derivation for multiple accounts from a single seed. An account is simply a deterministic identifier from which multiple addresses can be further derived. 
+
+The library also allows consumers to assign a meaningful alias to each account.
+
+### Multi-Account Approach
+
+The multi-account approach is used to create an account for each individual user. The created accounts can then be linked to the internal user IDs as an account alias, which are distinctly separated.
+
+### Single Account Approach
+
+The single account approach allows for just one account and creates addresses for each user. The associated addresses are then linked to the internal user IDs and store who owns which address in the database. Most exchanges are more familiar with the single account approach and find it easier to use, implement, and backup.
+
+## Implementation Guide
+
+This guide explains how to use the Wallet Library to successfully implement into an exchange.
+
+Features of the Wallet Library:
+- Secure seed management.
+- Account management (with multiple accounts and multiple addresses).
+- Confirmation monitoring.
+- Deposit address monitoring.
+- Backup and restore functionality.
+
+## How Does it Work?
+
+The Wallet Library is a stateful package with a standardized interface for developers to build applications involving value transactions. It offers abstractions to handle payments and can optionally interact with Stronghold for seed handling, seed storage, and state backup.
+
+For further reference, you can read our [wallet documentation here](https://wiki.iota.org/wallet.rs/welcome).
+
+The following examples cover the *multi-account approach* using the `NodeJS` binding:
+
+1. Set up the Wallet Library.
+2. Create an account for each user.
+3. Generate a User Address to Deposit Funds
+4. Listen to events.
+5. Check the user balance.
+6. Enable withdrawals.
+
+:::note
+
+If you are looking for other languages, please read the [wallet library overview](https://wiki.iota.org/wallet.rs/welcome).
+
+:::
+
+Since all `wallet.rs` bindings are based on core principles provided by the `wallet.rs` library, the outlined approach is very similar regardless of the programming language of your choice.
+
+### 1. Set up the Wallet Library
+First, you should install the components that are needed to use `wallet.rs` and the binding of your choice; it may vary a bit from language to language. In the case of the `NodeJs` binding, it is straightforward since it is distributed via the `npm` package manager. We also recommend you use `dotenv` for password management.
```

Comment:
```suggestion
First, you should install the components that are needed to use `wallet.rs` and the binding of your choice; it may vary a bit from language to language. In the case of the [NodeJs binding](../../getting_started/nodejs.md), it is straightforward since it is distributed via the `npm` package manager. We recommend that you use `dotenv` for password management.
```